com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/model/BooleanParameterConverter.java

```diff
@@ -0,0 +1,21 @@
+package com.sap.cloud.lm.sl.cf.web.api.model;
+
+import java.text.MessageFormat;
+
+import com.sap.cloud.lm.sl.common.SLException;
+
+public class BooleanParameterConverter implements ParameterConverter {
+
+    @Override
+    public Object convert(Object value) {
+        String parameterValue = String.valueOf(value);
+        if ("true".equals(parameterValue)) {
+            return true;
+        } else if ("false".equals(parameterValue)) {
+            return false;
+        } else {
+            throw new SLException(MessageFormat.format("Parameter is not boolean {0}", value));
```

Comment:
Do we need this exception? If not, we could make Boolean.valueOf?
How stupid would be to think that "opusum" = false?
Something else, we could implement a logic which is for validation of those parameters and then the conversion to happen?